src/components/Button/Button.tsx

```diff
@@ -75,12 +76,17 @@ export interface ButtonProps {
   onFocus?: (event: React.FocusEvent<HTMLButtonElement>) => void;
   /** On Button Blur callback */
   onBlur?: (event: React.FocusEvent<HTMLButtonElement>) => void;
+  /** Make the right side of the button flat (no rounded corners) */
   rightFlat?: boolean;
+  /** Make the right side of the button flat (no rounded corners) */
   leftFlat?: boolean;
+  /** Disables the click animation */
   preventClickAnimation?: boolean;
+  /** Removes the padding in the side of the button */
   noSidePadding?: boolean;
   /** default color for text color in ON_PRIMARY_COLOR kind (should be any type of css color (rbg, var, hex...) */
   defaultTextColorOnPrimaryColor?: string;
+  /** id to att for the data-test attribute */
```

Comment:
this description is a bit not clear. maybe: 
```suggestion
  /** unique identifier added to the button element for testing purposes */
```
or somethign like this?